Symbology/qgis/Confinement/constriction_ratio.qml

```diff
@@ -0,0 +1,479 @@
+<!DOCTYPE qgis PUBLIC 'http://mrcc.com/qgis.dtd' 'SYSTEM'>
+<qgis minScale="100000000" styleCategories="AllStyleCategories" hasScaleBasedVisibilityFlag="0" simplifyMaxScale="1" simplifyDrawingTol="1" simplifyAlgorithm="0" version="3.18.2-Zürich" maxScale="0" simplifyLocal="1" readOnly="0" labelsEnabled="0" simplifyDrawingHints="1">
+  <flags>
+    <Identifiable>1</Identifiable>
+    <Removable>1</Removable>
+    <Searchable>1</Searchable>
+    <Private>0</Private>
+  </flags>
+  <temporal enabled="0" durationUnit="min" endExpression="" durationField="" startField="" endField="" mode="0" accumulate="0" startExpression="" fixedDuration="0">
+    <fixedRange>
+      <start></start>
+      <end></end>
+    </fixedRange>
+  </temporal>
+  <renderer-v2 type="graduatedSymbol" forceraster="0" symbollevels="0" graduatedMethod="GraduatedColor" enableorderby="0" attr="Constriction_Ratio">
+    <ranges>
+      <range render="true" upper="0.000000000000000" lower="0.000000000000000" label="Not constricted" symbol="0"/>
+      <range render="true" upper="1.000000000000000" lower="0.000000000000000" label="Constricted" symbol="1"/>
```

Comment:
This should be something similar to confinement ratio as in NOT binary.  There is value in having constrictions highlighted discretely, but those are generally smaller than the total reach length and come from the Confinement Raw stuff (i.e. both).